go.mod

```diff
@@ -83,10 +83,15 @@ require (
 	golang.org/x/text v0.3.7
 	golang.org/x/tools v0.1.9
 	gonum.org/v1/gonum v0.11.0
-	google.golang.org/protobuf v1.27.1
+	google.golang.org/protobuf v1.28.0
 	gopkg.in/guregu/null.v4 v4.0.0
 )
 
+require (
+	github.com/codegangsta/negroni v1.0.0 // indirect
+	gopkg.in/guregu/null.v2 v2.1.2 // indirect
+)
```

Comment:
Let's combine these indirect require blocks in to one. I don't know why go mod does this :thinking: